applications/Unity.GrantManager/src/Unity.GrantManager.Application/ApplicationForms/ApplicationFormVersionAppService.cs

```diff
@@ -59,31 +60,41 @@ public override async Task<ApplicationFormVersionDto> GetAsync(Guid id)
             return dto;
         }
 
-        public async Task<bool> InitializePublishedFormVersion(dynamic chefsForm, Guid applicationFormId)
+        private static JToken? GetFormVersionToken(dynamic chefsForm) {
+            if (chefsForm == null) return null;
+            JObject formObject = JObject.Parse(chefsForm.ToString());
+            if (formObject == null) return null;
+
+#pragma warning disable CS8600
```

Comment:
can remove this by marking the JToken versionsToken as JToken? versionsToken

Reply:
Removed - fixed